demo/index.html

```diff
@@ -14,7 +14,7 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-  <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+  <script src="https://unpkg.com/@webcomponents/webcomponentsjs@2/webcomponents-loader.js"></script>
```

Comment:
Didn't think we'd be updating demos to use wc-loader.js instead of -bundle.js

Reply:
I think loader is better as a demo should be what people should be doing irl, no?

Reply:
Did we settle on `../node_modules/...`?

Reply:
That's what we'd eventually like to have, but it depends on https://github.com/unpkg/unpkg.com/issues/74 to work through webcomponents.org, and we didn't hear back about the timing there yet. This approach of fully qualifying the URL will work locally and on webcomponents.org, so it seemed like the best way to unblock for now. We can always update them later if things change.